src/main/kotlin/racing_car/domain/race/RaceGame.kt

```diff
@@ -0,0 +1,22 @@
+package racing_car.domain.race
+
+import racing_car.domain.car.Car
+
+class RaceGame(private val carList: List<Car>) {
+
+    init {
+        require(carList.isNotEmpty())
+    }
```

Comment:
`init`구문을 사용해 객체 초기화 및 검증을 잘 나타내주신 것 같아요👍
RageGame 클래스를 사용하는 사용자들에게 조금 더 자세한 예외 메시지를 제공하는 건 어떨까요?
```suggestion
    init {
         require(carList.isNotEmpty()) { "게임에 참가할 자동차가 없습니다." }
     }
```